.circleci/config.yml

```diff
@@ -623,80 +623,94 @@ jobs:
             docker push audius/<< parameters.repo >>:$(git rev-parse HEAD)
 
 workflows:
-  # test master at midnight daily
-  test-nightly:
-    jobs:
-      - test-libs:
-          force: true
-      - test-contracts:
-          force: true
-      - test-eth-contracts:
-          force: true
-      - test-creator-node:
-          force: true
-      - test-discovery-provider:
-          force: true
-      - test-identity-service:
-          force: true
-      - test-solana-programs:
-          force: true
-    triggers:
-      - schedule:
-          cron: '0 5 * * *'
-          filters:
-            branches:
-              only: /(^master$)/
-
   # test, build and push all commits
   test-build-and-push:
     jobs:
-      - test-libs
+      - test-libs:
+          name: test-libs
       - docker-build-and-push:
           name: build-libs
           repo: libs
           requires:
             - test-libs
 
-      - test-contracts
+      - test-contracts:
+          name: test-contracts
       - docker-build-and-push:
           name: build-contracts
           repo: contracts
           requires:
             - test-contracts
 
-      - test-eth-contracts
+      - test-eth-contracts:
+          name: test-eth-contracts
       - docker-build-and-push:
           name: build-eth-contracts
           repo: eth-contracts
           requires:
             - test-eth-contracts
 
-      - test-creator-node
+      - test-creator-node:
+          name: test-creator-node
       - docker-build-and-push:
           name: build-creator-node
           repo: creator-node
           requires:
             - test-creator-node
 
-      - test-discovery-provider
+      - test-discovery-provider:
+          name: test-discovery-provider
       - docker-build-and-push:
           name: build-discovery-provider
           repo: discovery-provider
           requires:
             - test-discovery-provider
 
-      - test-identity-service
+      - test-identity-service:
+          name: test-identity-service
       - docker-build-and-push:
           name: build-identity-service
           repo: identity-service
           requires:
             - test-identity-service
 
-      - test-solana-programs
+      - test-solana-programs:
+          name: test-solana-programs
       - docker-build-and-push:
           name: build-solana-programs
           repo: solana-programs
-          # requires:
-          #   - test-solana-programs
+          requires:
+            - test-solana-programs
 
       - test-mad-dog-e2e
+
+  # test master at midnight daily
+  test-nightly:
+    jobs:
+      - test-libs:
+          name: test-libs-nightly
```

Comment:
so, this reuses https://github.com/AudiusProject/audius-protocol/pull/1577/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R629, and is called `test-libs-nightly`? and ultimately removes the -1 and -2 suffixes?

Reply:
Yes, that is correct, so our nightly tests should be appended by `-nightly` moving forward

Reply:
cool! we can check to see next day that the nightly tests ran with the suffix `-nightly`

Reply:
https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/10138/workflows/4c49848e-9d9d-4966-b8a2-5cfb7d9486cb/jobs/85104